src/Helper/Quote.php

```diff
@@ -109,16 +109,42 @@ public function __construct(
     public function getValidatedCoupons()
     {
         $coupons = [];
+        $returnInvalidCartRuleCoupon = false;
```

Comment:
I think the `=` signs need to align, I cannot remember for sure, but I think this project styles are like that

Reply:
done